entities/Base.java

```diff
@@ -0,0 +1,59 @@
+package entities;
+
+import java.util.Date;
+
+public abstract class Base {
+
+    //Attributes
+
+    private Integer id;
```

Comment:
Use `Long` instead of `Integer`. As restaurant entities should grown in a level greater than the maximum value of an `Integer`